### PR TITLE
Gunicorn edit

### PR DESCRIPTION
**Pull request by ucheNkadiCode**
Removed Gunicorn specific debug directions. We now have a compose debug file that allows our users to volume map the debugger into a container. 

Updated Docker Compose Python directions with clearer information. 
Note: Will likely be out for the rest of the year starting Dec 14th, so feel free to make any edits or you can wait until I am able to make the necessary changes.

### CODE REVIEW

File: docs/containers/docker-compose.md [hunk 1]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
 
-1. Right-click on the `docker-compose.debug.yml` file and choose **Compose Up**.
+1. If you already have a valid Dockerfile, we highly recommend running the command **Docker: Add Docker Compose Files to Workspace**. This will create a `docker-compose.yml` file and also a `docker-compose.debug.yml` which volume maps and starts the Python debugger in the container. If you do not have a Dockerfile already, we recommend running **Docker: Add Docker Files to Workspace** and selecting **Yes** to include Docker Compose files.
 
-1. When you attach to a service that exposes an HTTP endpoint that returns HTML, the web browser doesn't open automatically. To open the app in the browser, choose the container in the sidebar, right-click and choose **Open in Browser**. If multiple ports are configured, you'll be asked to choose the port.
+    > Note: By default, when using **Docker: Add Docker Files to Workspace**, choosing the Django and Flask options will scaffold a Dockerfile configured for Gunicorn. Follow the instructions [here](/docs/containers/quickstart-python.md#gunicorn-modifications-for-djangoflask-apps
```
Comment (ghogen) [inline docs/containers/docker-compose.md:135]:
```suggestion
    > **Note**: By default, when using **Docker: Add Docker Files to Workspace**, choosing the Django and Flask options will scaffold a Dockerfile configured for Gunicorn. Follow the instructions [here](/docs/containers/quickstart-python.md#gunicorn-modifications-for-djangoflask-apps
```

File: docs/containers/docker-compose.md [hunk 2]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
+   ![Screenshot - Open in Browser](images/compose/docker-compose-open-in-browser.png)
+
+    You're now debugging your running app in the container.
 
 ### .NET
 
 1. On the **Debug** tab, choose the **Configuration** dropdown, choose **New Configuration** and select the `Docker Attach` configuration template **.NET Core Docker Attach (Preview)**.
 
-1. VS Code tries to copy `vsdbg` from the host machine to the target container using a default path. You can also provide a path to an existing instance of `vsdbg` in the **Attach** configuration.
+2. VS Code tries to copy `vsdbg` from the host machine to the target container using a default path. You can also provide a path to an existing instance of `vsdbg` in the **Attach** configuration.
```
Comment (ghogen) [inline docs/containers/docker-compose.md:171]:
```suggestion
1. VS Code tries to copy `vsdbg` from the host machine to the target container using a default path. You can also provide a path to an existing instance of `vsdbg` in the **Attach** configuration.
```
using 1 in app steps means automatic numbering

File: docs/containers/docker-compose.md [hunk 3]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
 
    ```json
     "netCore": {
         "debuggerPath": "/remote_debugger/vsdbg"
     }
    ```
 
-1. When done editing the **Attach** configuration, save `launch.json`, and select your new launch configuration as the active configuration. In the **Debug** tab, find the new configuration in the **Configuration** dropdown.
+3. When done editing the **Attach** configuration, save `launch.json`, and select your new launch configuration as the active configuration. In the **Debug** tab, find the new configuration in the **Configuration** dropdown.
```
Comment (ghogen) [inline docs/containers/docker-compose.md:179]:
```suggestion
1. When done editing the **Attach** configuration, save `launch.json`, and select your new launch configuration as the active configuration. In the **Debug** tab, find the new configuration in the **Configuration** dropdown.
```

File: docs/containers/docker-compose.md [hunk 4]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
 
-1. Right-click on the `docker-compose.debug.yml` file and choose **Compose Up**.
+4. Right-click on the `docker-compose.debug.yml` file and choose **Compose Up**.
```
Comment (ghogen) [inline docs/containers/docker-compose.md:181]:
```suggestion
1. Right-click on the `docker-compose.debug.yml` file and choose **Compose Up**.
```

File: docs/containers/docker-compose.md [hunk 5]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
 
-1. When you attach to a service that exposes an HTTP endpoint that returns HTML, the web browser doesn't open automatically. To open the app in the browser, choose the container in the sidebar, right-click and choose **Open in Browser**. If multiple ports are configured, you'll be asked to choose the port.
+5. When you attach to a service that exposes an HTTP endpoint that returns HTML, the web browser doesn't open automatically. To open the app in the browser, choose the container in the sidebar, right-click and choose **Open in Browser**. If multiple ports are configured, you'll be asked to choose the port.
```
Comment (ghogen) [inline docs/containers/docker-compose.md:183]:
```suggestion
1. When you attach to a service that exposes an HTTP endpoint that returns HTML, the web browser doesn't open automatically. To open the app in the browser, choose the container in the sidebar, right-click and choose **Open in Browser**. If multiple ports are configured, you'll be asked to choose the port.
```

File: docs/containers/docker-compose.md [hunk 6]
```diff
@@ -128,43 +128,65 @@ For debugging Python with Docker Compose, first read [How to debug your app with
...
 
-1. Launch the debugger in the usual way. From the **Debug** tab, choose the green arrow (**Start** button) or use `kb(workbench.action.debug.start)`.
+6. Launch the debugger in the usual way. From the **Debug** tab, choose the green arrow (**Start** button) or use `kb(workbench.action.debug.start)`.
 
    ![Screenshot of starting debugging](images/compose/docker-compose-attach.png)
 
-1. If you try to attach to a .NET Core app running in a container, you'll see a prompt ask to select your app's container.
+7. If you try to attach to a .NET Core app running in a container, you'll see a prompt ask to select your app's container.
```
Comment (ghogen) [inline docs/containers/docker-compose.md:189]:
```suggestion
1. If you try to attach to a .NET Core app running in a container, you'll see a prompt ask to select your app's container.
```

**Review by ghogen — APPROVED** [review]
Approved with suggestions

**Comment by ucheNkadiCode** [comment]
> Approved with suggestions

#sign-off 
I've submitted another PR to update the list numbers and committed the change to make "Note" bold.

**Pull request merged**